README.md

```diff
@@ -0,0 +1,45 @@
+# kwiz-dotfiles
+
+This repository holds the configuration files for `prettier` and `esling` that should be used in every project.
+
+## General
+
+**Note:** make sure you have a `.npmrc` with the following content:
```

Comment:
Also mention that one needs to login with a Github token locally.

Reply:
done, thanks!